Resources/Prototypes/Stray/Entities/Clothing/Under/pants.yml

```
@@ -12,6 +12,18 @@
       state: icon
     - type: TemperatureProtection
       coolingCoefficient: 0.80
+    - type: Food
+      requiresSpecialDigestion: true
+    - type: SolutionContainerManager
+      solutions:
+        food:
+          maxVol: 30
+          reagents:
+          - ReagentId: Fiber
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)